nhanes-blood-pressure/examining-blood-pressure.ipynb

```diff
@@ -91,7 +91,14 @@
     "\n",
     "Probably one of the simplest measurements of body function to collect is heart rate. There are [multiple locations on the body that can be used to measure heart rate](https://www.healthline.com/health/how-to-check-heart-rate), but the easiest is the radial artery in the wrist:\n",
     "\n",
-    "![Radial HR measurement](https://s32917.pcdn.co/wp-content/uploads/2020/02/VHFC0134_How_do_I_measure_my_heart_rate_image1.jpeg)\n",
+    "![Radial HR measurement](https://github.com/callysto/data-viz-of-the-week/raw/main/nhanes-blood-pressure/img/VHFC0134_How_do_I_measure_my_heart_rate_image1.jpeg)\n",
```

Comment:
Since the image file is local now, you can use the local file path instead of the url:

`![Radial HR measurement](img/VHFC0134_How_do_I_measure_my_heart_rate_image1.jpeg)`

Reply:
Done! Thanks for the feedback